### PR TITLE
Login tweaks

### PR DESCRIPTION
**Pull request by akshaaatt**
- A user friendly Login Activity with some codebase cleanup

### CODE REVIEW

**Review by amCap1712 — CHANGES_REQUESTED** [review]
Can you recheck and confirm that the creator of the asset hsa been attributed on the About Page? Other than that, LGTM. Thanks.

**Comment by akshaaatt** [comment]
@amCap1712 I actually intend to prepare a list of all the assets which we have used including icons, animations, etc with their links and authors to a text file and add it to the codebase so that we have everything systematically. I intend to this as soon as we merge everything because when we are attributing on the about page, we only have to add an author name so without the links to verify(from our end) it is difficult to do so.

Finally all I say is, we merge right now and I add a new branch on that master to take care of all the attributions because even the animations we have been using before 2021 need to be attributed.

**Pull request merged**
